ajax/libs/videojs-wavesurfer/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "videojs-wavesurfer",
+  "version": "2.0.1",
+  "description": "video.js plugin that adds a navigable waveform for audio and video files.",
+  "filename": "videojs.wavesurfer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/collab-project/videojs-wavesurfer.git"
+  },
+  "keywords": [
+    "waveform",
+    "audio",
+    "video",
+    "wavesurfer",
+    "videojs",
+    "videojs-plugin",
+    "player"
+  ],
+  "author": "Collab",
+  "license": "MIT",
+  "homepage": "https://collab-project.github.io/videojs-wavesurfer/",
+  "npmName": "videojs-wavesurfer",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
```

Comment:
Let's modify the npm auto-update config as the followings.
Because this pattern will get all the files before v1.2.7, and this is useful to front-end developer to choose more versions.
The current pattern will get the files after v1.3.0.
```js
  "npmName": "videojs-wavesurfer",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    },
    {
      "basePath": "",
      "files": [
        "videojs.wavesurfer*"
      ]
    }
  ]
```

Thank you.

Reply:
@sufuf3 if the non-dist basePath part is for the old version only, I won't suggest to use the flexible pattern, just use more static one, we don't want to have unexpected behavior in future.

Reply:
OK!
Got it!
Thank you.